public/docs/ts/latest/cli-quickstart.jade

```diff
@@ -122,157 +110,263 @@ figure.image-display
   and modified our first component. That's about all we'd expect to do for a "Hello, World" app.
```

Comment:
Replace "we" in these these final sentences. 

Suggested phrasing:
--

That's about all you'd expect to do in a "Hello, World" app.

You're ready to take the [Tour of Heroes Tutorial](./tutorial) and build a small application that demonstrates the great things you can build with Angular.

Or you can stick around a bit longer to learn about the files in your brand new project.